modules/dfpAdServerVideo.js

```diff
@@ -158,6 +158,9 @@ function getCustParams(bid, options) {
 
   const optCustParams = deepAccess(options, 'params.cust_params');
   let customParams = Object.assign({},
+    { hb_uuid: bid && bid.videoCacheKey },
```

Comment:
can you add a comment here with a link to the issue so folks looking at the code will understand?